src/search-manager/FilterByPublished.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  Badge,
+  Form,
+  Menu,
+  MenuItem,
+} from '@openedx/paragon';
+import { FilterList } from '@openedx/paragon/icons';
+import SearchFilterWidget from './SearchFilterWidget';
+import { useSearchContext } from './SearchManager';
+import { PublishStatus } from './data/api';
+
+/**
+ * A button with a dropdown that allows filtering the current search by publish status
+ */
+const FilterByPublished: React.FC<Record<never, never>> = () => {
+  const {
+    publishStatus,
+    publishStatusFilter,
+    setPublishStatusFilter,
+  } = useSearchContext();
+
+  const clearFilters = React.useCallback(() => {
+    setPublishStatusFilter([]);
+  }, []);
```

Comment:
Selecting a Published Status filter doesn't show the "clear filters" link?

Need to modify [`canClearFilters`](https://github.com/open-craft/frontend-app-authoring/blob/dvz/filter-by-published/src/search-manager/SearchManager.ts#L147-L152).